actionpack/test/controller/render_test.rb

```diff
@@ -100,6 +100,17 @@ def conditional_hello_with_record
     end
   end
 
+  def conditional_hello_with_array_of_records
+    ts = Time.now.utc.beginning_of_day
+
+    record = Struct.new(:updated_at, :cache_key).new(ts, "foo/123")
+    old_record = Struct.new(:updated_at, :cache_key).new(ts - 1.day, "bar/123")
```

Comment:
```suggestion
    record = Struct.new(:updated_at, :cache_key).new(Time.now.utc.beginning_of_day, "foo/123")
    old_record = Struct.new(:updated_at, :cache_key).new(Time.now.utc.beginning_of_day.yesterday, "bar/123")
```

Wonder if `yesterday` works here.